rustls/src/msgs/handshake.rs

```diff
@@ -1592,6 +1592,9 @@ impl Codec for ECParameters {
     }
 }
 
+/// This type combines a [`SignatureScheme`] and a signature payload produced with that scheme.
+///
+/// [`SignatureScheme`]: SignatureScheme
```

Comment:
I think you don't need the explicit link target in this case? IIRC it's only necessary when the target is not available from the enclosing scope.

Reply:
Oh neat. That's handy. I tested it out locally and it worked the way you described. Force pushed a fix.